bindings/python/setup.py

```diff
@@ -72,10 +72,12 @@
     LIBRARY_FILE = "libcapstone.so"
     STATIC_LIBRARY_FILE = 'libcapstone.a'
```

Comment:
This code could probably be simplified to use the name only once. But it's probably better for another PR

Reply:
Yes. The whole thing is tracked in https://github.com/capstone-engine/capstone/issues/2445